site/docs/education/egeria-dojo/developer/topic-cataloguing-provider-skeleton.md

```diff
@@ -22,7 +22,7 @@ Paste this code between the curly braces of the `TopicCataloguingProvider` class
     private static final String connectorQualifiedName = "EgeriaDojo:IntegrationConnector:Topics:TopicCataloguingConnector";
     private static final String connectorDisplayName   = "Topic Cataloging Integration Connector";
     private static final String connectorDescription   = "Connector maintains a list of KafkaTopic assets associated with an Apache Kafka event broker.";
-    private static final String connectorWikiPage      = "https://odpi.github.io/egeria-docs/connectors/integration/integration-connector/";
+    private static final String connectorWikiPage      = "https://egeria-project.org/concepts/integration-connector";
```

Comment:
This change moves the link from pointing to an entry in the connector catalog to the description of the integration connector concept. There should be a new page for this connector in egeria-docs/connectors/integration called topic-cataloguing-integration-connector.md that describes this connector. The link would then be:

"https://egeria-project.org/concepts/topic-cataloguing-integration-connector"